jetty9/src/com/thoughtworks/go/server/Jetty9Server.java

```diff
@@ -87,6 +88,8 @@ public void configure() throws Exception {
         server.setHandler(handlers);
         performCustomConfiguration();
         server.setStopAtShutdown(true);
+        SessionManager sessionManager = webAppContext.getSessionHandler().getSessionManager();
```

Comment:
This is inconsistent with how `setCookieExpirePeriod` is setup, can we make it consistent, otherwise looks good to merge.